packages/wrangler/e2e/helpers/normalize.ts

```diff
@@ -61,7 +61,7 @@ function removeUUID(str: string) {
  * Remove the Wrangler version/update check header
  */
 function removeVersionHeader(str: string): string {
-	const header = str.match(/----+\n/);
+	const header = str.match(/⛅️ wrangler .*\n----+\n/);
```

Comment:
Why the needs for this change?

Reply:
`pnpm` includes a lot of hyphens in it's output, which was causing the header not to be removed properly. This just makes the matching stricter.